app/components/table/component.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Story } from '@storybook/react/types-6-0';
+import Button from 'components/button';
+import Icon from 'components/icon';
+import STAR_SVG from 'svgs/ui/star.svg?sprite';
+import Table, { TableProps, TableRow, LabelValue } from './component';
+
+export default {
+  title: 'Components/Table',
+  component: Table,
+  argTypes: {},
+};
+
+const Template: Story<TableProps> = ({ ...args }: TableProps) => (
+  <Table {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  headers: [
+    {
+      label: 'Best',
+      id: 'best',
+      customCell: (row: LabelValue) => {
```

Comment:
You are not receiving a row here. You should receive 2 arguments, value & row. Row means all the data related to that row